README.md

```diff
@@ -1,17 +1,22 @@
 # FastQC-nf bioinformatics workflow
 
 ## Description
-FastQC-nf is a pipeline for quality control in both illumina short reads and Pacbio long reads. It's written in nextflow. 
+FastQC-nf is a pipeline for quality control in both illumina short reads and Pacbio long reads. 
+It's written in nextflow workflow. 
 
+## Run Scripts
+For main.nf
+```
+nextflow run main.nf
+```
 
-## User guide
+For main-1.nf
+```
+nextflow run -e.mode=ciao main-1.nf
```

Comment:
What is the reasoning behind using `-e.mode`? I've never seen this before. Can you instead define a parameter at the top of `main.nf` called hello and apply it to the workflow with `nextflow run main.nf --hello ciao`